schemas/generate_address_space.py

```diff
@@ -3,34 +3,37 @@
 xmlparser.py is a requirement. it is in opcua folder but to avoid importing all code, developer can link xmlparser.py in current directory
 """
 import sys
+import os
+import datetime
 import logging
 # sys.path.insert(0, "..")  # load local freeopcua implementation
 #from opcua import xmlparser
 import opcua.common.xmlparser as xmlparser
 
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
os is a "heavy" library, we may should use pathlib for such cases.

Reply:
@oroulet mentiond it in asyncua but i was more used to os.path but yes we can!

Reply:
this is not part of core library so not so important